postcss.config.cjs

```diff
@@ -2,6 +2,8 @@ module.exports = {
   plugins: [
     require('postcss-nested'),
     require('postcss-preset-env')({ stage: 1 }),
-    require('autoprefixer')
+    require('autoprefixer'),
+    require('tailwindcss/nesting')('postcss-nesting'),
```

Comment:
Is `postcss-nesting` needed? There is already a `nested` above lol

Reply:
Yes, it is needed. Tailwind throws warning if it missing

Reply:
OK - what about getting rid of the other one?